### PR TITLE
Update create-sharepoint-sites-and-add-users-with-powershell.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a note to the document stating that the resource quota works only on classic sites and not on modern sites.

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/3126

File: microsoft-365/enterprise/create-sharepoint-sites-and-add-users-with-powershell.md [hunk 1]
```diff
@@ -39,6 +39,9 @@ The PowerShell cmdlet imports the .csv file and pipes it to a loop inside the cu
 
 ### Create a .csv file
 
+> [!Note]
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/create-sharepoint-sites-and-add-users-with-powershell.md]:
```suggestion
> [!NOTE]
```

File: microsoft-365/enterprise/create-sharepoint-sites-and-add-users-with-powershell.md [hunk 2]
```diff
@@ -39,6 +39,9 @@ The PowerShell cmdlet imports the .csv file and pipes it to a loop inside the cu
...
+> The resource quota parameter works only on Classic sites. If you use this parameter on a modern site, you may receive a warning message that it has been deprecated.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/create-sharepoint-sites-and-add-users-with-powershell.md]:
```suggestion
> The resource quota parameter works only on classic sites. If you use this parameter on a modern site, you may receive a warning message that it has been deprecated. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@JoeDavies-MSFT Please merge this PR. Thanks.

**Pull request merged**